Tools/SetupUnityPackage/UnityUtils/FileHelper.cs

```diff
@@ -17,13 +17,13 @@
 ////////////////////////////////////////////////////////////////////////////
 
 
-using System;
 using UnityEngine;
 
 namespace UnityUtils
 {
     public static class FileHelper
     {
+        [RuntimeInitializeOnLoadMethod]
```

Comment:
Now that we're not using JNI, do we still need this?

Reply:
I tested it and it appears that we still need that.